src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: $white-color;
+  @include flex-center;
+  flex-direction: column;
```

Comment:
Let's add border and border radius to the card.

Reply:
As it is in figma

Reply:
Hope everything is done 😅